app/routes/api/jobs.py

```diff
@@ -3,20 +3,23 @@
 """
 
 import os
-
+import os.path
 from datetime import datetime
+import fasteners
 import hashlib
-from marshmallow import ValidationError
-from sqlalchemy import and_
-from flask import Response, request
-from flask import send_file
+import json
 import logging
-
+from pathlib import Path
```

Comment:
I think we should either use `pathlib` or `os.path` to handle paths, but not both.

Reply:
I have been switching some of my code to pathlib, which I think is a bit easier and more obvious. However, that does not mean that every module in SEAMM needs to use one or the other across the entire stack. Probably better not to use both in one module, I do agree.